src/main/resources/db/changelog/changes/00000000000002_update_schema-20240704090100_changelog.yml

```diff
@@ -6,6 +6,6 @@ databaseChangeLog:
         - addColumn:
             columns:
               - column:
-                  name: email
+                  name: email_address
```

Comment:
In the User entity code, this is still referenced as `@Column(name = "email")`, so was it correct previously?

Reply:
Indeed. Fixed!